spec/features/govbox_registration_spec.rb

```diff
@@ -1,16 +1,204 @@
 require 'rails_helper'
 
 RSpec.feature 'Govbox registration', type: :feature do
-  scenario 'User registers for GovBox' do
-    visit '/sluzby/govbox'
-    within('#registration') do
-      click_on 'Mám záujem o GovBox'
+  background do
+    stub_request(:get, "#{ENV.fetch('AUTOFORM_URL')}/api/corporate_bodies/search").
+      with(query: hash_including(q: "cin:50 158 635", private_access_token: ENV.fetch('AUTOFORM_PRIVATE_ACCESS_TOKEN')))
+      .to_return(status: 200, body: file_fixture('autoform_api_response.json').read)
+  end
+
+  context 'User registers for govbox' do
```

Comment:
takto sa to pise (skontroluj to prosim aj napriec tymto repom)

```suggestion
  context 'User registers for GovBox' do
```